### PR TITLE
Remove bigtableadmin artifact

### PR DESCRIPTION
**Pull request by igorbernstein2**
Removes the google-bigtable-admin artifact (it was merged into the main bigtable artifact). Kokoro configs have been updated to stop referencing this artifact.

### CODE REVIEW

**Comment by kolea2** [comment]
Just to confirm as it's a little hard to see in the diffs, does this PR remove the entire bigtable-admin module?

File: versions.txt [hunk 1]
```diff
@@ -13,7 +13,6 @@ google-cloud-bigquery:1.62.0:1.62.1-SNAPSHOT
 google-cloud-bigquerydatatransfer:0.80.0-beta:0.80.1-beta-SNAPSHOT
 google-cloud-bigquerystorage:0.80.0-beta:0.80.1-beta-SNAPSHOT
 google-cloud-bigtable:0.80.0-alpha:0.80.1-alpha-SNAPSHOT
-google-cloud-bigtable-admin:0.80.0-alpha:0.80.1-alpha-SNAPSHOT
```
Comment (kolea2) [inline versions.txt]: what about proto-google-cloud-bigtable-admin-v2:0.45.0:0.45.1-SNAPSHOT and grpc?

Comment (igorbernstein2) [reply versions.txt]: Those should stay.  Only the gapic client was merged

Comment (igorbernstein2) [reply versions.txt]: The generated protos & grpc stubs, are still published separately:
https://github.com/googleapis/google-cloud-java/blob/master/google-cloud-clients/google-cloud-bigtable/synth.py#L46-L47

While the gapic library is merged:
https://github.com/googleapis/google-cloud-java/blob/master/google-cloud-clients/google-cloud-bigtable/synth.py#L45

**Comment by igorbernstein2** [comment]
Yep, the entire module is removed, you can see how the tree looks here:
https://github.com/igorbernstein2/google-cloud-java/tree/remove-bigtable/google-cloud-clients

**Comment by sduskis** [comment]
@igorbernstein2, can you please resolve the conflicts?

**Comment by igorbernstein2** [comment]
Fixed, the test failure is unrelated

**Pull request merged**